src/index.js

```diff
@@ -3,8 +3,20 @@ import express from 'express';
 import mongoose from 'mongoose';
 
 import { DATABASE_URL, PORT } from './util/env';
-import { typeDefs } from './typeDefs';
-import { resolvers } from './resolvers';
+import { typeDefs } from './graphql/typeDefs';
+import { resolvers } from './graphql/resolvers';
+import SpotifyAPI from './graphql/datsources/spotify';
+
+// Setup dataSources our resolvers need
+const dataSources = () => ({
+  SpotifyAPI,
+});
+
+// Function that sets up global context for resolvers.
```

Comment:
thank you for the comments. This helps.

Reply:
Comments really helped in seeing what is going on.